runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -4508,6 +4511,25 @@ void J9::X86::TreeEvaluator::transactionalMemoryJITMonitorEntry(TR::Node
       cg->stopUsingRegister(counterReg);
    }
 
+void
+J9::X86::TreeEvaluator::monitorEnterOrExitForValueTypeClass(
```

Comment:
Shouldn't this have the verb generate in the name?

Reply:
changed the name to `generateCheckForValueTypeMonitorEnterOrExit`.